assets/dev/scss/editor/panel/controls/_select.scss

```diff
@@ -39,4 +39,14 @@
 			pointer-events: none;
 		}
 	}
+
+	.elementor-control-field {
```

Comment:
@hein-obox Check nesting.

```
.elementor-control {
    
    &-input-wrapper {}
    
    &-control-field {}
}
```

Older CSS and your new CSS can be nested inside `.elementor-control{}`

Also, this nesting is not necessary:

```
&.elementor-control-field-select-small {

	.elementor-control-input-wrapper {
		max-width: 80px;
	}
}
```

Change to:

```
&.elementor-control-field-select-small .elementor-control-input-wrapper {
	max-width: 80px;
}
```

Reply:
Refactoring implemented.

@rodolphebertozzo Please check this out :)